tardis/gui.py

```diff
@@ -8,32 +41,61 @@
 from matplotlib.figure import *
 from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg as FigureCanvas
 from matplotlib.backends.backend_qt4 import NavigationToolbar2QT as NavigationToolbar
-import os
-if os.environ.get('QT_API', None) is None:
+if os.environ.get('QT_API', None)=='pyqt':
     from PyQt4 import QtGui, QtCore
-else:
+elif os.environ.get('QT_API', None)=='pyside':
     from PySide import QtGui, QtCore
-    
+else:
+    raise ImportError('QT_API was not set! Please exit the IPython console'+
+        ' and at the bash prompt use : \n\n export QT_API=pyside \n or \n'+
```

Comment:
Yes `\` is the last resort but `+` is bad ! If you have a better solution to make it multiline then its good but `+` should be avoided for the multiline purpose as far as I know.